string path
sensor_msgs/Image image